lstchain/reco/dl1_to_dl2.py

```diff
@@ -731,6 +730,9 @@ def apply_models(dl1,
 
     # gammaness is the prediction probability for the first class (0)
     dl2['gammaness'] = probs[:, 0]
+    col = list(classifier.classes_).index(0)
```

Comment:
Nice that you get the col automtaically.. but the line above still uses the hard-coded index

Reply:
@SeiyaNozaki could you move line 733 up one line and use col in 732?
After that we can merge I think

Reply:
sorry I forgot to implement @maxnoe's comment.. now implemented!